Rules.md

```diff
@@ -256,28 +256,50 @@ At any point in the game, you are allowed to give coins to another player.
 
 # Initial setup
 
-## Apply scenario
+## Initial supply
 
-Choose a [scenario](Scenarios.md) for the desired number of players *at random* and apply it.
-
-## Coins
-
-* Every player gets 2 coins in every region they own
 * Every player gets 3 coins in their supply
 * Every player gets command tokens:
     * Invest x 2
     * Dig trenches x 1
     * Move x 3
     * Train units x 2
 
-## Initial seeding
+## Beginner
+
+Choose a [scenario](Scenarios.md) for the desired number of players *at random*.
+
+* Every player gets 2 coins in every region they own
+* Remove one start unit from the non-strategic start region of every player.
```

Comment:
I think this should be a separate PR. I may agree, but I think it should also be in the scenarios.